tests/test_client.py

```diff
@@ -4,8 +4,10 @@
 import sys
 from datetime import datetime
 
+import mock
 import pkg_resources
 import pytest
+import requests
```

Comment:
unused import?

Reply:
requests is on line 267 for `requests.exceptions.ReadTimeout`